### PR TITLE
STORY-4757_international_static_number_creation: added promo_type and…

### PR DESCRIPTION
**Pull request by Vicentecarrillo**
… promo_number_looks_like fields to promo numbers NI docs
@mikeweaver

### CODE REVIEW

File: source/api_documentation/network_integration/promo_numbers/index.rst [hunk 1]
```diff
@@ -110,6 +110,20 @@ When you POST a promo number, we return the promo_number in the response body. T
...
+    - string (optional)
+    - For international campaigns that support local or mobile numbers. If not passed then promo numbers will default to the organizational default. (US/CA campaigns will continue to use local_center for local numbers)
+
+      "Local"
+
+      "Mobile"
+
+  * - promo_number_looks_like
+    - string (optional)
+    - For international local promo number creation, a string phone number to try to match can be included. The first five digits will be used. (ex. “805”, “80561”, "8051234567")
```
Comment (mikeweaver) [inline source/api_documentation/network_integration/promo_numbers/index.rst]: The last example contradicts the text: "The first five digits will be used."

File: source/api_documentation/network_integration/promo_numbers/index.rst [hunk 2]
```diff
@@ -110,6 +110,20 @@ When you POST a promo number, we return the promo_number in the response body. T
     - array of strings
     - an array of stringified limiters on the boundaries of where to look for local numbers given as npa or npa nxx pairs (ex. [“805”, “805612”])
 
+  * - promo_type
+    - string (optional)
+    - For international campaigns that support local or mobile numbers. If not passed then promo numbers will default to the organizational default. (US/CA campaigns will continue to use local_center for local numbers)
+
+      "Local"
+
+      "Mobile"
```
Comment (collin-mcgrath) [inline source/api_documentation/network_integration/promo_numbers/index.rst:119]: The NI API won't actually accept `'Local'` and `'Mobile'`. They need to be lower case. Found while QAing STORY-4950. We should either fix here or in the platform.

Comment (Vicentecarrillo) [reply source/api_documentation/network_integration/promo_numbers/index.rst:119]: @jebentier is resolving this in an in progress ticket, so we are good leaving the documentation as is

**Comment by Vicentecarrillo** [comment]
@mikeweaver

File: source/api_documentation/network_integration/promo_numbers/index.rst (hunk 2 above)
Comment (jebentier) [reply source/api_documentation/network_integration/promo_numbers/index.rst:119]: resolved.

**Pull request merged**